src/phasorpy/phasor.py

```diff
@@ -1934,6 +1936,150 @@ def phasor_from_fret_acceptor(
     )
 
 
+def phasor_to_principal_plane(
+    real: ArrayLike,
+    imag: ArrayLike,
+    /,
+    *,
+    reorient: bool = True,
+) -> tuple[NDArray[Any], NDArray[Any], NDArray[Any]]:
+    """Return multi-harmonic phasor coordinates projected onto principal plane.
+
+    The Principal Component Analysis (PCA) is used to project
+    multi-harmonic phasor coordinates onto a plane, along which
+    coordinate axes the phasor coordinates have the largest variations.
+
+    The transformed coordinates are not phasor coordinates. However, the
+    coordinates can be used in visualization and cursor analysis since
+    the transformation is affine (preserving collinearity and ratios
+    of distances).
+
+    Parameters
+    ----------
+    real : array_like
+        Real component of multi-harmonic phasor coordinates.
+        The first axis is the frequency dimension.
+        If less than 2-dimensional, size-1 dimensions are prepended.
+    imag : array_like
+        Imaginary component of multi-harmonic phasor coordinates.
+        Must be of same shape as `real`.
+    reorient : bool, optional, default: True
+        Reorient coordinates for easier visualization.
+        The projected coordinates are rotated and scaled, such that
+        the center lies in same quadrant and the projection
+        of [1, 0] lies at [1, 0].
+
+    Returns
+    -------
+    x : ndarray
+        X-coordinates of projected phasor coordinates.
+        If not `reorient`, this is the coordinate on the first principal axis.
+        The shape is ``real.shape[1:]``.
+    y : ndarray
+        Y-coordinates of projected phasor coordinates.
+        If not `reorient`, this is the coordinate on the second principal axis.
+    transformation_matrix : ndarray
+        Affine transformation matrix used to project phasor coordinates.
+        The shape is ``(2, 2 * real.shape[0])``.
+        The matrix can be used to project multi-harmonic phasor coordinates,
+        where the first axis is the frequency::
+
+            x, y = numpy.dot(
+                numpy.vstack(
+                    real.reshape(real.shape[0], -1),
+                    imag.reshape(imag.shape[0], -1),
+                ),
+                transformation_matrix,
+            ).reshape(2, *real.shape[1:])
+
```

Comment:
Maybe add reference to the tutorial?

Also a reference to the paper you cited in the PR description can also be helpful  to understand it's applications.